tests/test_runtime.py

```
@@ -0,0 +1,61 @@
+import os
+import tempfile
+import unittest
+from pathlib import Path
+from unittest.mock import patch
+
+import runtime
+
+
+class RuntimeHelpersTests(unittest.TestCase):
+    def test_resolve_fake_data_dir_prefers_railway_volume_over_default_data_path(self):
+        repo_root = Path("/tmp/simple-safer-server")
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/simple-safer-server"

(S108)